prov/efa/src/rdm/rxr_env.c

```diff
@@ -221,26 +207,78 @@ void rxr_env_define()
 			"The maximum number of bytes that will be eagerly sent by inflight messages uses runting read message protocol (Default 307200).");
 }
 
-/**
- * @brief Initialize the variables in rxr_env.
- * This function uses the g_device_list initialized in
- * efa_device_list_initialize(), thus it must be called
- * after efa_device_list_initialize().
- */
-void rxr_env_initialize()
+
+/* This function specifies the default value for using the
+ * device's RDMA capability. It also gives the help message
+ * about whether the environment variable FI_EFA_USE_DEVICE_RDMA
+ * can be set or not. */
```

Comment:
We use doxygen style comment for function header.

Example: https://github.com/ofiwg/libfabric/blob/c1f7b8b6e4d7ad25c3f6e38ca49b276ed309ef00/prov/efa/src/rdm/rxr_op_entry.c#L356